agent/verify.go

```
@@ -0,0 +1,167 @@
+	}
+	return ""
+}
+
+// hasMakeTestTarget returns true if the Makefile at path contains a "test:" target.
+// Only the first 1 KB is scanned to keep it fast.
+func hasMakeTestTarget(path string) bool {
+	f, err := os.Open(path)
+	if err != nil {
+		return false
+	}
+	defer f.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `f.Close` is not checked

(errcheck)